src/layout/_template.njk

```diff
@@ -43,9 +44,12 @@
 
 {# Page columns #}
 {% set pageColNumber = pageConfig.pageColNumber | default("12") %}
-{% set pageColClasses = pageConfig.pageColClasses if pageConfig.pageColClasses else '' %}
-{% set mainClasses = pageConfig.mainColClasses if pageConfig.mainColClasses else '' %}
-
+{% set pageColClasses = pageConfig.pageColClasses if pageConfig.pageColClasses else 
+    '' %}
+{% set mainClasses = pageConfig.mainColClasses if pageConfig.mainColClasses else 
+    '' %}
+{% set theme = pageConfig.theme if pageConfig.theme else 
```

Comment:
I think the theme param name should be more specific. Maybe `themeColour`? Also this param will need to be documented in the service manual

Reply:
I have changed this to themeColour :) and yes the documentation on the service manual will need to be updated :)